examples/next-app-router/src/app/globals.css

```diff
@@ -0,0 +1,20 @@
+@tailwind base;
```

Comment:
I noticed the visual editor's component hover labels are shorter when running this app compared to the Test App.

It looks like the tailwind base styling is doing that. When I commented out this line the label matches the size in the Test App.  I'm not sure it's a big deal but wanted to mention it.

![Screenshot 2024-06-05 at 12 15 44 PM](https://github.com/contentful/experience-builder/assets/8539634/fe8735a4-9686-4393-8186-35261a624e90)